hack/build-cross.sh

```diff
@@ -35,6 +35,7 @@ OS_GOFLAGS="${OS_GOFLAGS:-} ${OS_IMAGE_COMPILE_GOFLAGS}" os::build::build_static
 OS_RELEASE_ARCHIVE="openshift-origin"
 OS_BUILD_PLATFORMS=("${platforms[@]}")
 os::build::place_bins "${OS_CROSS_COMPILE_BINARIES[@]}"
+echo "${OS_GIT_COMMIT}" > "${OS_LOCAL_RELEASEPATH}/.commit"
```

Comment:
Need to distinguish between dirty and not here.  If you're tree state is dirty this would be wrong